ui/lib/core/addon/components/alert-banner.js

```diff
@@ -13,36 +11,28 @@ import layout from '../templates/components/alert-banner';
  * ```
  *
  * @param {String} type=null  - The banner type. This comes from the message-types helper.
- * @param {String} [secondIconType=null] - If you want a second icon to appear to the right of the title. This comes from the message-types helper.
- * @param {Object} [progressBar=null] - An object containing a value and maximum for a progress bar. Will be displayed next to the message title.
+ * @param {String} [bannerType=alert] - Defaults to 'alert', can be used to specify an alert banner's test selector.
+ * @param {Boolean} [marginless=false] - If true changes the container class to margin: 0.
  * @param {String} [message=null] - The message to display within the banner.
+ * @param {Object} [progressBar=null] - An object containing a value and maximum for a progress bar. Will be displayed next to the message title.
+ * @param {Boolean} [showLoading=false] - Shows a loading icon to the right of the title.
  * @param {String} [title=null] - A title to show above the message. If this is not provided, there are default values for each type of alert.
- * @param {String} [bannerType=alert] - Defaults to 'alert', can be used to specify an alert banner's test selector
- *
+ * @param {Boolean} [yieldWithoutColumn=false] - If true, do not show message or title, just yield with no formatting.
+ * @param {String} [learnLink=null] - Display a DocLink pointing to this path.
+ * @param {String} [learnLinkMessage=null] - Show a learn link message to the right of a learn-link icon. Displays below the main message.
  */
 
-export default Component.extend({
-  layout,
-  type: null,
-  message: null,
-  title: null,
-  secondIconType: null,
-  progressBar: null,
-  yieldWithoutColumn: false,
-  marginless: false,
-  classNameBindings: ['containerClass'],
-  bannerType: 'alert',
-
-  containerClass: computed('type', 'marginless', function () {
-    const base = this.marginless ? 'message message-marginless ' : 'message ';
-    return base + messageTypes([this.type]).class;
-  }),
+export default class AlertBanner extends Component {
+  get bannerType() {
```

Comment:
Re: comment above I think we can remove this and just have 'alert' directly in the template